### PR TITLE
Split Falco audit example into own page

### LINKED ISSUE #15963: Split Falco audit example into own page
**This is a Feature Request**

**What would you like to be added**
Make a new page that includes the Falco details from https://k8s.io/docs/tasks/debug-application-cluster/audit/#use-falco-to-collect-audit-events
Hyperlink from https://k8s.io/docs/tasks/debug-application-cluster/audit/ to the new page.

**Why is this needed**
https://k8s.io/docs/tasks/debug-application-cluster/audit/ is a long page, long enough that readers might get lost or give up.

**Comments**

[Falco](https://landscape.cncf.io/selected=falco) is a CNCF project.

### PR DESCRIPTION
**Pull request by medmedchiheb**
Fixes  #15963

### CODE REVIEW

File: content/en/docs/tasks/debug-application-cluster/audit.md [hunk 1]
```diff
@@ -491,109 +491,6 @@ plugin which supports full-text search and analytics.
...
-[fluentd_plugin_management_doc]: https://docs.fluentd.org/v1.0/articles/plugin-management
-[logstash]: https://www.elastic.co/products/logstash
-[logstash_install_doc]: https://www.elastic.co/guide/en/logstash/current/installing-logstash.html
-[kube-aggregator]: /docs/concepts/api-extension/apiserver-aggregation
-[falco_website]: https://www.falco.org
-[falco_k8s_audit_rules]: https://github.com/falcosecurity/falco/blob/master/rules/k8s_audit_rules.yaml
-[falco_ka_docs]: https://falco.org/docs/event-sources/kubernetes-audit
-[falco_installation]: https://falco.org/docs/installation
-[falco_helm_chart]: https://github.com/helm/charts/tree/master/stable/falco
+[Use Falco to collect audit events](/docs/tasks/debug-application-cluster/falco)
```
Comment (sftim) [inline content/en/docs/tasks/debug-application-cluster/audit.md]: I recommend creating a new whatsnext section, similar to https://github.com/kubernetes/website/blame/9f1d574aa9f5df5f69f63141e96f3ee6daa905e6/content/en/docs/tasks/debug-application-cluster/debug-service.md#L641

File: content/en/docs/tasks/debug-application-cluster/falco.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
+---
+reviewers:
+- soltysh
+- sttts
+- ericchiang
+content_template: templates/concept
+title: Webhook Collector Examples
```
Comment (sftim) [inline content/en/docs/tasks/debug-application-cluster/falco.md]: nit: this title looks wrong.

Comment (medmedchiheb) [reply content/en/docs/tasks/debug-application-cluster/falco.md]: What would be the title!

Comment (sftim) [reply content/en/docs/tasks/debug-application-cluster/falco.md]: How about “Auditing with Falco”?

**Comment by medmedchiheb** [comment]
ready for merge

File: content/en/docs/tasks/debug-application-cluster/audit.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/tasks/debug-application-cluster/audit.md]: Also it's worth matching the title of the link to the target.

**Comment by sftim** [comment]
/lgtm
Looking at #15892 this change seems OK. It's about doing a Kubernetes thing (cluster audit) using a CNCF project to help with that.

**Comment by aimeeu** [comment]
I know this has already been merged.... but Falco has its own [documentation](https://falco.org/docs/).This content might be better located in the Falco documentation and then linked to from the Kubernetes documentation.

**Pull request merged**